MapboxNavigation.xcodeproj/xcshareddata/xcschemes/Example-CarPlay.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "C53F2EDE20EBC95600D9798F"
-               BuildableName = "Example-CarPlay.app"
+               BuildableName = "Example.app"
```

Comment:
Looks strange to me, why `BuildableName` was changed to `Example.app`?

Reply:
See https://github.com/mapbox/mapbox-navigation-ios/pull/2832#discussion_r584000207 and https://github.com/mapbox/mapbox-navigation-ios/pull/2832#discussion_r607423578. The naming collision is unfortunate but appears to be unavoidable now.